src/org/linphone/LinphoneActivity.java

```diff
@@ -1277,8 +1279,7 @@ public boolean onKeyDown(int keyCode, KeyEvent event) {
 		return super.onKeyDown(keyCode, event);
 	}
 }
-
 interface ContactPicked {
 	void setAddresGoToDialerAndCall(String number, String name, Uri photo);
 	void goToDialer();
-}
+}
```

Comment:
Just revert this file. You've added a trailing LF at the end of the file. It's easier just to revert this file.